### DISCUSSION TITLE
Can I Ask Khoj to search the internet?

### DISCUSSION THREAD

**Discussion by Lastofthefirst** [khoj-ai/khoj#374] [General]
This looks great. I'd love to be able to use the same tool to also search documentation. Can I ask it to search online documentation as well? I saw another discussion mentioning tradeoffs between content that is searchable and the speed or ux of the tool. Couldn't this be addressed with toggles, some way of indicating that if I write or do xyz I want to search only a certain format?

**Comment by debanjum** [comment]
Hi @Lastofthefirst, a few others folks have also asked for similar stuff. We're trying to brainstorm the best way to address this in [this discussion](https://github.com/khoj-ai/khoj/discussions/423). Please do share your thoughts there as well
